File: src/main/java/employeetech/com/employee/manager/model/Student.java
package employeetech.com.employee.manager.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
public class Student implements Serializable {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(nullable = false, updatable = false)
    private Long id;

    private String name;
    private String course;
    private String email;
    private Integer age;
}

File: src/main/kotlin/employeetech/com/employee/manager/model/Student.kt
package employeetech.com.employee.manager.model

import jakarta.persistence.*
import java.io.Serializable

@Entity
data class Student(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(nullable = false, updatable = false)
    val id: Long? = null,

    val name: String? = null,
    val course: String? = null,
    val email: String? = null,
    val age: Int? = null
) : Serializable
